src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -183,18 +259,28 @@ const ActionsPageEvent = ({
       <div className={styles.content}>
         <div className={styles.text}>
           <FormattedMessage
-            id={
-              eventName === ColonyAndExtensionsEvents.ColonyMetadata
-                ? getColonyMetadataMessageDescriptorsIds(
-                    ColonyAndExtensionsEvents.ColonyMetadata,
-                    getColonyMetadataChecks,
-                  )
-                : 'event.title'
-            }
+            id={getEventTitleMessageDescriptor}
             values={{
               ...values,
               fromDomain: values?.fromDomain?.name,
               toDomain: values?.toDomain?.name,
+              oldColor: (
+                <ColorTag
+                  color={
+                    Number(previousDomainMetadata?.domainColor) ||
+                    Color.LightPink
+                  }
+                />
+              ),
+              domainColor: (
+                <ColorTag color={Number(domainColor) || Color.LightPink} />
+              ),
+              // Fallback to current name/description when there is no previous metadata obj
+              oldDescription:
+                previousDomainMetadata?.domainPurpose || domainPurpose,
+              oldName: previousDomainMetadata?.domainName || domainName,
+              domainPurpose,
+              domainName,
```

Comment:
The logic here is all good, but the display is not.

Currently, you show the most current domain name, which is not always correct, especially if you go back through multiple edits.

It should just show the old and new domain names relevant to the action that was taken

Here's an example:

![Screenshot from 2021-02-03 00-10-07](https://user-images.githubusercontent.com/1193222/106669407-58025a80-65b4-11eb-9263-ba3c19241d56.png)

I've changed the name from three to four, but this shows the domain name as "SAME", which is the most recent name I've set to the domain.

This is not correct. This should just say "Changed team's name from three to four"

While I understand this was the copy spec that product handed over, I think the spec/copy is wrong and should be changed. 

Please raise this problem with Product.

Reply:
Im not sure what exact problem you see with this? It will be hard for me to raise this problem with Product without explaining what is the problem. So can you give me more details?

Reply:
Ahh i think a got it, so you want to update copies to show just "from to" values?

Reply:
So the problem here is that we show the most recent domain name, even for actions that happened in the past.

In my example "SAME NAME" is the latest domain name I've changed. This should only be shown on the _latest_ domain edit action.

The previous one, should have the old name, and the one before that the even-older name, ... and so on.

If you want we can jump on a call and I can show you the problem live

Reply:
Exactly. Since it makes more sense that way